### ISSUE TITLE
Need clarifications on how sending TXs work.

### ISSUE THREAD

**Issue by markrufino** [reaster/cardano_wallet_sdk#6]
Just curious as to how this works since based on my understanding this wallet is not running a node. Correct me if I'm wrong does this work by just serializing a TX, and then using blockfrost.io to do the heavylifting? Just wanted to know because I'm planning to use this as a personal tool to pass ADA around, I want to understand it deeper. Thanks for making this thing! :)

**Comment by reaster** [comment]
Most mobile and web wallets use third party services for accessing the blockchain. However, self custodial wallets hold the private keys, so the third party service can not sign your transactions. Consequently, the wallet must build the transaction, sign it and attach the signature to the transaction in the correct binary format (not trivial). The third party service, will check that the inputs match the outputs, validate the signature and submit it for inclusion in the blockchain, but can not change it any way.  
FYI There is still a bug in the signing code and I probably won't have time to fix it right away. I'm working full time and taking the Plutus Pioneer Course. This project still needs a lot of work to achieve production quality. When that happens, I expect there will be a lot of interest in the Flutter community.

**Comment by haskell-monad** [comment]
> 大多数移动和网络钱包使用第三方服务来访问区块链。 但是，自托管钱包持有私钥，因此第三方服务无法签署您的交易。 因此，钱包必须构建交易，对其进行签名并以正确的二进制格式（并非微不足道）将签名附加到交易中。 第三方服务将检查输入是否与输出匹配，验证签名并将其提交以包含在区块链中，但不能以任何方式更改它。 仅供参考，签名代码中仍然存在错误，我可能没有时间立即修复它。 我正在全职工作并参加 Plutus Pioneer 课程。 该项目仍需要大量工作才能达到生产质量。 当这种情况发生时，我预计 Flutter 社区会引起很大的兴趣。

So far, the signature problem has been fixed. I want to use this library to develop real applications. There seems to be no similar library available.

**Comment by reaster** [comment]
Yes. Signature now work. However, this is a prototype SDK, it was not meant to be used in real applications. The plan is to rewrite it over the next 8 months to be to bring it up to production quality.

**Comment by haskell-monad** [comment]
> 

thank you for your hard work, come on
